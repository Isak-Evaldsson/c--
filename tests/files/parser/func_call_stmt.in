int func() {
    func1();
    func2(1);
    func3(a, b, 1);
}