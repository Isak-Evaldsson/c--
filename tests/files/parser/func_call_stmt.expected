Prototype: func
    FunCall: func1
    FunCall: func2
        LiteralExpr: 1
    FunCall: func3
        VarUseExpr: a
        VarUseExpr: b
        LiteralExpr: 1
